compiler/passes/checkParsed.cpp

```diff
@@ -399,11 +399,10 @@ checkFunction(FnSymbol* fn) {
     }
   }
 
-#if 0 // Do not issue the warning yet.
+  // Do not issue the warning yet.
```

Comment:
The comment seems like it was referring to the `#if 0` itself, rather than the code within it